test/e2e/pages/vl-map-select-action.page.js

```diff
@@ -1,10 +1,10 @@
 const VlMapSelectAction = require('../components/vl-map-select-action');
-const VlMap = require('../components/vl-map');
 const VlMapLayer = require('../components/vl-map-layer');
-const {Page, Config} = require('vl-ui-core').Test;
+const {Config} = require('vl-ui-core').Test;
```

Comment:
lijkt alsof we de Config niet meer gebruiken maar ik kan er over gekeken hebben

Reply:
De `Config` wordt nog gebruikt in de `load` functie.